### PR TITLE
re-fetch to ensure br.Bridge has latest attributes

### PR DESCRIPTION
**Pull request by yaocw2020**
issue: #9 
solution: re-fetch bridge at the end of process of ensuring bridge

### CODE REVIEW

File: pkg/controller/vlan/bridge_vlan_controller.go [hunk 1]
```diff
@@ -161,6 +161,7 @@ func (c *BridgeVLANController) configBridgeNetwork(setting *NetworkSetting) erro
 	// nic is nil if setting.NIC is an empty string
 	// return addresses to configured nic if nic is empty
 	if nic == nil && configuredNIC != nil {
+		klog.Infof("Return address because nic is nil and configured nic is not nil, configured nic: %s", configuredNIC.Attrs().Name)
```
Comment (guangbochen) [inline pkg/controller/vlan/bridge_vlan_controller.go:164]: start with a small case to keep it consistent.

File: pkg/controller/vlan/bridge_vlan_controller.go [hunk 2]
```diff
@@ -171,6 +172,7 @@ func (c *BridgeVLANController) configBridgeNetwork(setting *NetworkSetting) erro
 	// - Yes，do nothing
 	// - No, make bridge return addresses to configured nic, lend addresses to bridge, return.
 	if nic.Attrs().MasterIndex != c.bridge.Index {
+		klog.Infof("Configure network because the master of nic %s is not bridge %s", nic.Attrs().Name, c.bridge.Name)
```
Comment (guangbochen) [inline pkg/controller/vlan/bridge_vlan_controller.go:175]: ditto

File: pkg/controller/vlan/bridge_vlan_controller.go [hunk 3]
```diff
@@ -193,6 +195,7 @@ func (c *BridgeVLANController) configBridgeNetwork(setting *NetworkSetting) erro
 		return fmt.Errorf("could not list addresses, error: %w, link: %s", err, c.bridge.Name)
 	}
 	if len(addrList) == 0 {
+		klog.Infof("Configure network because bridge %s has no address", c.bridge.Name)
```
Comment (guangbochen) [inline pkg/controller/vlan/bridge_vlan_controller.go:198]: ditto

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**